plugins/org.jkiss.dbeaver.ext.altibase/src/org/jkiss/dbeaver/ext/altibase/model/AltibaseReplicationItem.java

```diff
@@ -14,46 +14,62 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package org.jkiss.dbeaver.ext.altibase.model;
 
 import org.jkiss.code.NotNull;
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.ext.altibase.AltibaseUtils;
 import org.jkiss.dbeaver.model.exec.jdbc.JDBCResultSet;
 import org.jkiss.dbeaver.model.impl.jdbc.JDBCUtils;
 import org.jkiss.dbeaver.model.meta.Property;
-import org.jkiss.utils.CommonUtils;
+import org.jkiss.dbeaver.model.runtime.DBRProgressMonitor;
+import org.jkiss.dbeaver.model.struct.DBSAlias;
+import org.jkiss.dbeaver.model.struct.DBSObject;
+
+/*
+ * ReplicationItem is a pair of replication tables consisting of local and remote. 
+ * Replication is possible not only at the table level but also at the table partition level.
+ * 
+ * A replication can have multiple replicationItems.
+ */
+public class AltibaseReplicationItem extends AltibaseObject<AltibaseReplication> implements DBSAlias {
 
-public class AltibaseReplicationItem extends AltibaseObject<AltibaseReplication> {
+    // Replication table name index
+    private static final int TBL_SCHEMA     = 0;
+    private static final int TBL_NAME       = 1;
+    private static final int TBL_PARTN      = 2; // partition
+    private static final int REPL_TBL_CNT   = TBL_PARTN + 1;
 
     private String tableOid;
-    
-    private String replObjFrom;
-    private String replObjTo;
+
+    private String[] replObjFrom = new String[REPL_TBL_CNT];
+    private String[] replObjTo = new String[REPL_TBL_CNT];
```

Comment:
Please store individual values separately instead of putting them into arrays. It doesn't improve code readability.

Reply:
I think everyone's preferences may vary. I'll modify it to fit the standards of the DBeaver product :)